pkgs/applications/networking/instant-messengers/hamsket/default.nix

```diff
@@ -21,7 +21,7 @@ let
     categories = [ "Network" ];
   });
 
-  appimageContents = appimageTools.extractType2 {
+  appimageContents = appimageTools.extract {
```

Comment:
This entire variable can probably go away.

Reply:
What do you mean, exactly?

Reply:
Hm, I thought you could do it via `src` but perhaps not.